pdfs/src/Pdf/FromLatex.hs

```diff
@@ -0,0 +1,206 @@
+
+module Pdf.FromLatex where
+
+import qualified Data.ByteString as BS
+import Path.IO
+import Path
+import System.Directory
+import System.IO.Temp
+import System.Process.Typed
+
+escapeInLatex :: Text -> Text
+escapeInLatex text = text
+
+type Locale = Text
+
+data Date = Date
+    { year  :: Int
+    , month :: Int
+    , day   :: Int
+    }
+
+dateToString :: Locale -> Date -> Text
+dateToString _ (Date y m d) = show d <> "." <> show m <> "." <> show y
+
+data StudentInfo = StudentInfo
+    { siName        :: Text
+    , siDateOfBirth :: Date
+    , siSections    :: [Section]
+    }
+
+data Section = Section
+    { sEducator :: Text
+    , sDiploma  :: Diploma
+    , sCourses  :: [Course]
+    }
+
+data Diploma = Diploma
+    { dEducatorUrl     :: Text
+    , dEducationPeriod :: (Int, Int)
```

Comment:
Could you prompt what this field is about?

Reply:
"Year from" and "year to". I can add or search for a `Range a` type here.

Reply:
Ah, right, I failed to interpret `period` word correctly.
Some `Range` would be pretty nice here yeah (that would diverge from @flyingleafe's version though :roll_eyes:)